static/backbone/js/api/embedded-mapplication.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Created by zmmachar on 12/11/14.
+ */
+define(["marionette",
+        "backbone",
+        "underscore",
+        "views/maps/basemap",
+        "lib/maps/data/viewLoader",
+        "lib/appUtilities",
+        "collections/projects",
+        "lib/maps/controls/georeferenceManager",
+        "jquery.bootstrap"
+    ],
+    function (Marionette, Backbone, _, BaseMap, ViewLoader, appUtilities, Projects, GeoreferenceManager) {
+        "use strict";
+
+        var Mapplication = new Marionette.Application();
+        _.extend(Mapplication, appUtilities);
+        Mapplication.setMode('view');
+
+        Mapplication.addRegions({
+            mapRegion: "#map_canvas",
+            sidebarRegion: "#panels"
+        });
+
+        Mapplication.navigate = function (route, options) {
+            options = options || {};
+            Backbone.history.navigate(route, options);
+        };
+
+        Mapplication.getCurrentRoute = function () {
+            return Backbone.history.fragment;
+        };
+
+        Mapplication.on("start", function () {
+            if (Backbone.history) {
+                Backbone.history.start();
+            }
+        });
+
+        Mapplication.addInitializer(function (options) {
+            options.projects = new Projects();
+            options.app = this;
+            var basemap = new BaseMap(options),
+                georeferenceManager = new GeoreferenceManager(options, basemap),
```

Comment:
The GeoreferenceManager is only useful for editing overlay geometries, so I'd drop it from this file (since this version of the Mapplication should be read only).